pipestat_example/looper_config_pipestat.yaml

```diff
@@ -0,0 +1,32 @@
+pep_config: ./project/project_config_pipestat.yaml # pephub registry path or local path
+output_dir: "./results"
+pipeline_interfaces:
+  sample:  ./pipeline/pipeline_interface.yaml
+  project: ./pipeline/pipeline_interface_project.yaml
+pipestat:
+  project:
+    project_name: test
+    sample_name: frog_1
+    schema_path: ./pipeline/pipestat_output_schema.yaml
+    results_file_path: "results.yaml"  #this is an absolute path for now
+    database:
+      dialect: postgresql
+      driver: psycopg2
+      name: pipestat-test
+      user: postgres
+      password: pipestat-password
+      host: 127.0.0.1
+      port: 5432
+  sample:
+    project_name: test
+    sample_name: frog_1
+    schema_path: ./pipeline/pipestat_output_schema.yaml
```

Comment:
this is the output schema for the other pipeline, so it should not be duplicated.